docs/serving/samples/hello-world/helloworld-java/pom.xml

```diff
@@ -14,7 +14,7 @@
 	<parent>
 		<groupId>org.springframework.boot</groupId>
 		<artifactId>spring-boot-starter-parent</artifactId>
-		<version>2.0.3.RELEASE</version>
+		<version>2.1.3.RELEASE</version>
```

Comment:
Why upgrade to 2.1.3 directly? What is the versioning guidance for this?

Reply:
it's the latest GA version.